packages/monorepo-builder/src/Finder/PackageComposerFinder.php

```diff
@@ -47,6 +57,12 @@ public function getPackageComposerFiles(): array
             ->exclude('node_modules')
             ->name('composer.json');
 
+        if ($this->packageDirectoriesExcludes !== []) {
```

Comment:
This is not needed, as it's always array

Reply:
fixed this